src/constants.ts

```diff
@@ -8,6 +8,9 @@
  * SPDX-License-Identifier: EPL-2.0
  **********************************************************************/
 
+// labels
+export const CHE_RELATED_COMPONENT_LABEL = 'app=che'
```

Comment:
Pls remove label usage. 
If am not sure about appropriate label

Reply:
I would like to keep the label. It gives better user experience, imo. But I agree, that the label itself should be changed